packages/palette/src/elements/BarChart/Bar.tsx

```diff
@@ -14,6 +14,7 @@ const LABEL_OFFSET = 10
 
 interface BarBoxProps {
   isHighlighted?: boolean
+  hasOnClick?: boolean
```

Comment:
Do we need this extra prop? Maybe instead we could check whether an 'onClick' callback was supplied?